contracts/base/Dispatcher.sol

```diff
@@ -242,6 +243,17 @@ abstract contract Dispatcher is Payments, V2SwapRouter, V3SwapRouter, V3ToV4Migr
                     }
 
                     (success, output) = address(V3_POSITION_MANAGER).call(inputs);
+                } else if (command == Commands.V4_POSITION_CALL) {
+                    bytes4 selector;
+                    assembly {
+                        selector := calldataload(inputs.offset)
+                    }
+
+                    if (selector != IPositionManager.modifyLiquidities.selector) {
```

Comment:
chatted on slack but i say remove this check

Reply:
yeah i agree 
someone could permit the UR over their v4 position and then call modifyLiquidities with the decrease command, causing anyone would be able to decrease/burn on their position since UR is approved. 
but i will document that they should only call modifyLiquidities to mint or increase liquidity since they should only use UR for migrating